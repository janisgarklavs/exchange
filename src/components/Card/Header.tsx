import React from "react";

const styleCX: React.CSSProperties = {
  display: "flex",
  flexDirection: "column",
  justifyContent: "center"
};

export const Header: React.SFC = ({ children }) => (
  <div style={styleCX}>{children}</div>
);
