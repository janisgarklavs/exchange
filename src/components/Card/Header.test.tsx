import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

import { Header } from "./Header";

describe("<Header />", () => {
  test("it renders correctly", () => {
    const component = shallow(<Header />);

    expect(toJson(component)).toMatchSnapshot();
  });
});
