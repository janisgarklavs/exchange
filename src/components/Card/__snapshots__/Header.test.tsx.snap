// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> it renders correctly 1`] = `
<div
  style={
    Object {
      "display": "flex",
      "flexDirection": "column",
      "justifyContent": "center",
    }
  }
/>
`;
